lib/honeybadger/plugins/shoryuken.rb

```diff
@@ -0,0 +1,46 @@
+require 'honeybadger/plugin'
+require 'honeybadger'
+
+module Honeybadger
+  module Plugins
+    module Shoryuken
+      class Middleware
+        def call(worker, queue, sqs_msg, body)
+          if sqs_msg.is_a?(Array)
```

Comment:
hi @ivanvc @joshuap 

I was wondering, maybe when it's a batch `Array`, we could notify it like `Honeybadger.notify(e, parameters: { batch: body })`, WDYT?

The only thing we wouldn't be able to precisely get is `ApproximateReceiveCount`, has each message would have one. Maybe we could skip this `ApproximateReceiveCount` checker if `is_a? Array`. Any thoughts?